### DISCUSSION TITLE
[FeatReq] GeoMap: Add Choropleth mode

### DISCUSSION THREAD

**Discussion by MXTRN0** [grafana/grafana#68027] [Dashboards, Panels & Visualizations]
A Choropleth "Mode" or view for the geoMap would be nice to have, as many have location data, that only includes only the country and having a random dot in the middle of the country is a bit misleading.

So a view where the whole country is colored would be very nice.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82687. Feel free to continue the discussion around this there. Thank you!
